tests/API_contract_tests.postman_collection.json

```diff
@@ -3593,13 +3603,243 @@
 			],
 			"description": "DSA that uploads data to their Tidepool Account, adds notes and shares data with their clinician in preparation for a clinic appointment.",
 			"protocolProfileBehavior": {}
+		},
+		{
+			"name": "Status Checks",
+			"item": [
+				{
+					"name": "auth status check",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "dc915f3c-cbe6-454b-8174-fad6045d7e68",
+								"exec": [
+									"pm.test(\"Status code is 200\", function () {\r",
```

Comment:
should we not just use our standardised check here?? 

then we can also assert no token is returned, and at least log the response time 

```
const assert = eval(globals.loadAsserts);
assert.common(200, 300, false, null,'');
```